src/NuGet/NuGetProjectPackUtil.csproj

```diff
@@ -20,9 +20,9 @@
   <Target Name="_InitializeNuspecProperties" BeforeTargets="GenerateNuspec" DependsOnTargets="InitializeSourceControlInformation">
     <PropertyGroup>
       <_Version></_Version>
-      <_Version Condition="'$(NuGetPackageKind)' == 'release'">$(NuGetReleaseVersion)</_Version>
-      <_Version Condition="'$(NuGetPackageKind)' == 'prerelease'">$(NuGetPreReleaseVersion)</_Version>
-      <_Version Condition="'$(NuGetPackageKind)' == 'perbuildprerelease'">$(NuGetPerBuildPreReleaseVersion)</_Version>
+      <_Version Condition="'$(NuGetPackageKind)' == 'release'">>$(VersionPrefix)</_Version>
```

Comment:
❓ Is the added `>` intentional? If so, I would prefer it be escaped (`&gt;`) to avoid this question in the future.

Reply:
Not intentional. Good catch.